src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -12,18 +11,69 @@ import {
   GEL_SPACING_DBL,
   GEL_SPACING_TRPL,
 } from '@bbc/gel-foundations/spacings';
-
+import Grid from '@bbc/psammead-grid';
```

Comment:
Should we not be using the `Grid` component from `/src/app/components/Grid`?

https://github.com/bbc/simorgh/blob/latest/src/app/components/Grid/index.jsx

Reply:
Not sure what this component is meant to do over and above importing `psammead-grid`?

Reply:
Just had a chat with @sareh about this, looks like the [Simorgh grid component](https://github.com/bbc/simorgh/blob/latest/src/app/components/Grid/index.jsx) is intended for use as a 'full page' grid, not a 'sub-grid' like we have here, in our case we need custom grid styling as it is used in the secondary column at full screen breakpoints then at the bottom of the story for smaller breakpoints.